docs/source/development/index.rst

```diff
@@ -14,30 +12,180 @@ The key components of AiiDAlab are:
+This is to ensure that these apps work as intended when installed.
+
+AiiDAlab and AiiDAlab Home app development
+------------------------------------------
+
+For the following instructions, it is assumed that any development is directly performed within the AiiDAlab Terminal environment.
+
+.. tip::
+
+    The steps to setup (and teardown) an AiiDAlab development workflow are automated within the `aiidalab-dev <https://github.com/aiidalab/aiidalab-dev>`__ package.
+
+Setup for front-end development (home app)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In your development environment (local or in the cloud), remove the symbolic link of aiidalab-home (``/home/aiida/apps/home``) and replace it with a link to a clone of your fork of the aiidalab-home app.
+Then commence development on a new branch:
+
+.. code-block:: console
+
+    ~$ git clone git@github.com:<USERNAME>/aiidalab-home.git
+    ~$ cd apps/
+    ~/apps$ rm home   # DO NOT USE `rm -r`!
+    ~/apps$ ln -s ~/aiidalab-home/ home
+
+Once satisfied with the change set, push it to your fork and create a
+(draft) pull request.
+This will ensure that the front-end tests are executed against the latest development version of aiidalab-docker-stack.
+Address all issues and then release a new version.
+
+Setup for back-end development (aiidalab package)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+First, replace the system home app with a local copy based on your fork as described in the instructions for development of the home app.
+The clone your fork of the aiidalab package into a location within user space of your choice (e.g. ``$HOME/aiidalab``) and create a symbolic link within the home app to that location:
+
+.. code-block:: console
+
+    ~$ git clone git@github.com:<USERNAME>/aiidalab.git
+    ~$ cd apps/home/
+    ~/apps/home$ ln -s $HOME/aiidalab/aiidalab
+
+In this way, you can test the function of the home app directly against
+your development version of aiidalab.
+
+Once satisfied with the change set, push it to your fork and create a
+(draft) pull request.
+This will ensure that the unit and integration tests are executed.
+
+How to make a new release (aiidalab and aiidalab-home)
+------------------------------------------------------
+
+#. Switch to ``develop`` branch.
+#. Make a new release branch e.g. ``release/1.0.0a2``.
+
+.. code-block:: console
+
+       $ git checkout -b release/1.0.0a2
+
+#. Increment ``__version__`` in ``__init__.py`` AND ``setup.json``
+   (setup.py).
+#. Commit the changes with the commit message: "Prepare release
+   1.0.0a2".
+#. Push the changes to GitHub.
+#. Create a PR from ``release/1.0.0a2`` branch to the ``master`` branch.
+#. Once the tests are passed, merge the PR via merge commit (do not use
+   rebase, it will recreate all the commits' sha!).
+#. Go to the command line and pull the ``master`` branch.
+#. Delete ``release/1.0.0a2`` branch locally.
+
+.. code-block:: console
+
+        $ git branch -d release/1.0.0a2
```

Comment:
I am not sure why, but in this case and in some other cases below the command is not highlighted as a code block.